packages/@sanity/color-input/sanity.json

```diff
@@ -0,0 +1,24 @@
+{
+  "paths": {
+    "source": "./src",
+    "compiled": "./lib"
+  },
+  "parts": [
```

Comment:
Not sure we need to expose these parts. Instead, we should implement the schema type part.
See https://github.com/sanity-io/plugin-template-chess-input/blob/master/template/sanity.json for an example

Reply:
👍